donation/views.py

```diff
@@ -13,18 +16,38 @@ def default(self, obj):
             return float(obj)
         return super().default(obj)
 
+def ong_required(function):
+    @wraps(function)
+    def wrapper(request, *args, **kwargs):
+        if request.user.ong.name.lower() == "asem" or request.user.ong.name.lower() == "videssur":
```

Comment:
If any ngo is required, then, you should delete this fucntion, because any superuser should have its ngo attached to it. I mean, I know we only work for Videssur and ASEM, and that is the main reason that I don't see this function necessary.